### PR TITLE
Update release-4.0.12.md

### PR DESCRIPTION
**Pull request by overvenus**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: releases/release-4.0.12.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ TiDB 版本：4.0.12
     + TiCDC
 
         - 若任务的暂停同步时间超过 1 天，再次启动该任务时需要二次确认 [#1497](https://github.com/pingcap/ticdc/pull/1497)
+        - 添加关于 Old Value 功能的监控面板 [#1571](https://github.com/pingcap/ticdc/pull/1571)
```
Comment (TomShawn) [inline releases/release-4.0.12.md:55]:
```suggestion
        - 为 Old Value 功能添加监控面板 [#1571](https://github.com/pingcap/ticdc/pull/1571)
```

File: releases/release-4.0.12.md [hunk 2]
```diff
@@ -119,6 +120,9 @@ TiDB 版本：4.0.12
 
         - 修复 `resolved ts` 时间乱序的问题 [#1464](https://github.com/pingcap/ticdc/pull/1464)
         - 修复由于网络不稳定而导致的表调度出错引发的数据丢失问题 [#1508](https://github.com/pingcap/ticdc/pull/1508)
+        - 修复 processor 停止是没有及时释放资源的问题 [#1547](https://github.com/pingcap/ticdc/pull/1547)
```
Comment (TomShawn) [inline releases/release-4.0.12.md:123]:
```suggestion
        - 修复终止 processor 时没有及时释放资源的问题 [#1547](https://github.com/pingcap/ticdc/pull/1547)
```

File: releases/release-4.0.12.md [hunk 3]
```diff
@@ -119,6 +120,9 @@ TiDB 版本：4.0.12
...
+        - 修复潜在的下游数据库链接泄露问题 [#1524](https://github.com/pingcap/ticdc/pull/1524)
```
Comment (TomShawn) [inline releases/release-4.0.12.md:124]: 能否和英文对齐？

File: releases/release-4.0.12.md [hunk 4]
```diff
@@ -119,6 +120,9 @@ TiDB 版本：4.0.12
...
+        - 修复在与 PD 通信异常可能发生的丢失数据表同步任务的问题 [#1540](https://github.com/pingcap/ticdc/pull/1540)
```
Comment (TomShawn) [inline releases/release-4.0.12.md:125]: 与英文对齐
```suggestion
        - 修复因 PD 抖动时多个 owner 共存可能导致数据表丢失的问题 [#1540](https://github.com/pingcap/ticdc/pull/1540)
```

File: releases/release-4.0.12.md (hunk 2 above)
Comment (qiancai) [reply releases/release-4.0.12.md:123]: suggestion updating "没有" to "不能“

Comment (TomShawn) [reply releases/release-4.0.12.md:123]: done

**Review by amyangfei — APPROVED** [review]

**Review by leoppro — APPROVED** [review]

**Pull request merged**